modules/apps/sync/sync-engine/src/main/java/com/liferay/sync/engine/file/system/SyncWatchEventProcessor.java

```diff
@@ -717,7 +728,33 @@ else if (eventType.equals(SyncWatchEvent.EVENT_TYPE_MODIFY)) {
 				}
 			}
 			else {
-				modifyFile(syncWatchEvent);
+				if (OSDetector.isWindows()) {
```

Comment:
This directly addresses the issue in SYNC-1713. Renaming files/folder w/ just the case causes a "EVENT_TYPE_MODIFY". I'm basically just detecting this situation and triggering a "RENAME" event instead.